src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/pubsub.yaml

```diff
@@ -103,7 +103,7 @@ services:
     lifecycle:
       run:
         windows:
-          powershell -command sleep 1
+          powershell -command sleep 100
```

Comment:
why are these changes needed?

Reply:
@tilo-chen

Reply:
If it was dead, why did we try to kill it? Sounds like something customers could run into too

Reply:
Right I should revert it. That actually covers more cases.
We do check `process.isAlive()` before the whole graceful kill. But the first AttachConsole can still fail with ERROR_ACCESS_DENIED.